flake.nix

```diff
@@ -1,4 +1,15 @@
 {
+  nixConfig = {
+    extra-substituters = [
+      "https://someone-llama-cpp.cachix.org" # FIXME: replace with an "official" one
```

Comment:
@ggerganov ought to be in control of this, I think, unless he'd rather delegate.

Reply:
Yes, I meant to replace these once @ggerganov has set up another cachix